pkgs/tools/misc/tmuxinator/default.nix

```diff
@@ -8,8 +8,8 @@ buildRubyGem rec {
   inherit ruby;
   name = "${gemName}-${version}";
   gemName = "tmuxinator";
-  version = "0.10.0";
-  source.sha256 = "199pq15qknpcafw8ryb9kk1jsrwnncg6k5l9d4n0nmms4knxlqlf";
+  version = "0.10.1";
+  source.sha256 = "0rjy2glqwbz07ci0snycq19myfczd2pry2iw4g0nqsw37wclm1vi";
```

Comment:
Can any of the dependencies listed below also be updated?

Reply:
Done
Sorry for the delay, I didn't see your message